packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -453,6 +472,14 @@ export class GarbageCollector implements IGarbageCollector {
             && !gcOptions.disableGC
         );
 
+        const currentVersion = pkgVersion;
+        const minimumVersion = this.mc.config.getString(trackGCBlobStateMinimumVersionKey);
+        const shouldTrackStateForVersion = meetsMinimumVersionRequirement(currentVersion, minimumVersion);
+
+        const trackGCBlobStateEnabled = this.mc.config.getBoolean(trackGCBlobStateKey) === true;
```

Comment:
nit: Use `this.mc.config.getBoolean(trackGCBlobStateKey) === true` directly in the line below. That will avoid creating an extra variable.